### PR TITLE
Ensure version checks content length

### PR DESCRIPTION
**Pull request by sivamu**

### CODE REVIEW

File: baseline.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "requests":
   [
-    {"path":"/version","perfTarget":{"category":"Static"}},
+    {"path":"/version","perfTarget":{"category":"Static"},"validation":{"contentType":"text/plain", "minLength":5}},
     {"path":"/index.html","perfTarget":{"category":"Static"},"validation":{"contentType":"text/html","contains":["<title>Swagger UI</title>","<div id=\"swagger-ui\">"]}},
```
Comment (bartr) [inline baseline.json:5]: I would put a maxLength on here as well

File: memory-baseline.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "requests":
   [
-    {"path":"/memory/version","perfTarget":{"category":"Static"}},
+    {"path":"/memory/version","perfTarget":{"category":"Static"},"validation":{"contentType":"text/plain", "minLength":5}},
     {"path":"/memory/index.html","perfTarget":{"category":"Static"},"validation":{"contentType":"text/html","contains":["<title>Swagger UI</title>","<div id=\"swagger-ui\">"]}},
```
Comment (bartr) [inline memory-baseline.json:5]: same

**Review by bartr — APPROVED** [review]
approved w/ changes

**Review by joaquinrz — APPROVED** [review]

**Comment by sivamu** [comment]
Closing this for now to discuss how to handle difference in ngsa and ngsa-java body formats.

**Pull request closed without merging**